File: api/Entities/Table.cs
namespace AiMenu.Api.Entities;

public class Table
{
    public Guid TableId { get; set; }
    public Guid RestaurantId { get; set; }
    public string Name { get; set; } = string.Empty;
    public string QrCodeValue { get; set; } = string.Empty;
    public bool IsActive { get; set; } = true;

    public Restaurant Restaurant { get; set; } = null!;
    public ICollection<Order> Orders { get; set; } = new List<Order>();
}

File: frontend/admin-web/src/types/table.ts
export type AdminTable = {
  tableId: string;
  restaurantId: string;
  name: string;
  menuUrl: string;
  qrCodeValue: string;
  isActive: boolean;
};

export type TableFormValues = {
  restaurantId: string;
  name: string;
  isActive: boolean;
};
